CP01CNPM/CP01CNPM_R00002_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP01CNPM-RII01-02-ADCPTG010,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/instruments/ADCP_sn*/*.000,CP01CNPM-RII01-02-ADCPTG010,recovered_inst,Available,
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00002/cg_data/stc1/irid2shore/stc_status.*_*.txt,CP01CNPM-SBS01-00-STCENG000,recovered_host,Available,

```

Comment:
There is no irid2shore folder in cg_data/stc1 in the Raw Data Repository.

Reply:
I believe this directory is excluded from the public (raw data archive) but I don't remember why. Could be that they may contain IP addresses and other potential sensitive info?

The files are on the cgpss/omsds and available to the ingestion process:
/home/ooiuser/DS/CP01CNPM/R00002/cg_data/stc1/irid2shore